src/SDK/SmartDetectorsExtensionsPackage/SmartDetectorsExtensionsPackage.nuproj

```diff
@@ -48,9 +48,6 @@
     <Dependency Include="Microsoft.Azure.Monitoring.SmartDetectors.RuntimeEnvironment.Contracts">
       <Version>0.13.1</Version>
     </Dependency>
-    <Dependency Include="Microsoft.CodeAnalysis.CSharp.Scripting">
```

Comment:
>Microsoft.CodeAnalysis.CSharp.Scripting [](start = 25, length = 39)

Our code does not uses this nuget anymore? I remember that it was used in the CreateContractAlert method